### PR TITLE
[minor] Typo/Grammatical corrections

### PR DESCRIPTION
**Pull request by PoojaChandak**
## What is the purpose of the change

*(Explain why we should have this change)*

## How was the change tested

*(Explain what tests did you do to verify the code change)*

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -166,13 +166,13 @@ public static class ExclamationBolt implements IRichBolt {
...
 
-The `cleanup` method is called when a Bolt is being shutdown and should cleanup any resources that were opened. There's no guarantee that this method will be called on the cluster: for example, if the machine the task is running on blows up, there's no way to invoke the method. The `cleanup` method is intended for when you run topologies in [local mode](Local-mode.html) (where a Storm cluster is simulated in process), and you want to be able to run and kill many topologies without suffering any resource leaks.
+The `cleanup` method is called when a Bolt is being shutdown and should cleanup any resources that were opened. There's no guarantee that this method will be called on the cluster: for example, if the machine the task is running on blows up, there's no way to invoke the method. The `cleanup` method is intended for when you run topologies in [local mode](Local-mode.html) (where a Storm cluster is simulated in the process), and you want to be able to run and kill many topologies without suffering any resource leaks.
```
Comment (Ethanlm) [inline docs/Tutorial.md:175]: "where a Storm cluster is simulated in the process" should be "where a Storm cluster is simulated in a process".

Local mode means a storm cluster runs in a same JVM process.

File: docs/Tutorial.md [hunk 2]
```diff
@@ -206,13 +206,13 @@ public static class ExclamationBolt extends BaseRichBolt {
 
 Let's see how to run the `ExclamationTopology` in local mode and see that it's working.
 
-Storm has two modes of operation: local mode and distributed mode. In local mode, Storm executes completely in process by simulating worker nodes with threads. Local mode is useful for testing and development of topologies. You can read more about running topologies in local mode on [Local mode](Local-mode.html).
+Storm has two modes of operation: local mode and distributed mode. In local mode, Storm executes completely in the process by simulating worker nodes with threads. Local mode is useful for testing and development of topologies. You can read more about running topologies in local mode on [Local mode](Local-mode.html).
```
Comment (Ethanlm) [inline docs/Tutorial.md:209]: the -> a

File: docs/Tutorial.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ A topology runs forever, or until you kill it. Storm will automatically reassign
 
 Storm uses tuples as its data model. A tuple is a named list of values, and a field in a tuple can be an object of any type. Out of the box, Storm supports all the primitive types, strings, and byte arrays as tuple field values. To use an object of another type, you just need to implement [a serializer](Serialization.html) for the type.
 
-Every node in a topology must declare the output fields for the tuples it emits. For example, this bolt declares that it emits 2-tuples with the fields "double" and "triple":
+Every node in topology must declare the output fields for the tuples it emits. For example, this bolt declares that it emits 2-tuples with the fields "double" and "triple":
```
Comment (Ethanlm) [inline docs/Tutorial.md:62]: should we keep "a"?

File: docs/Tutorial.md [hunk 4]
```diff
@@ -206,13 +206,13 @@ public static class ExclamationBolt extends BaseRichBolt {
...
+Storm has two modes of operation: local mode and distributed mode. In local mode, Storm executes completely in the process by simulating worker nodes with threads. Local mode is useful for testing and development of topologies. You can read more about running topologies in local mode on [Local mode](Local-mode.html).
 
 To run a topology in local mode run the command `storm local` instead of `storm jar`.
 
 ## Stream groupings
 
-A stream grouping tells a topology how to send tuples between two components. Remember, spouts and bolts execute in parallel as many tasks across the cluster. If you look at how a topology is executing at the task level, it looks something like this:
+A stream grouping tells a topology how to send tuples between two components. Remember, spouts and bolts execute in parallel as many tasks across the cluster. If you look at how topology is executing at the task level, it looks something like this:
```
Comment (Ethanlm) [inline docs/Tutorial.md:215]: should we keep "a"?

**Comment by PoojaChandak** [comment]
Hi, I have made changes as suggested in the review comments. Kindly check.

**Pull request merged**